Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Insolation"
 uuid = "e98cc03f-d57e-4e3c-b70c-8d51efe9e0d8"
 authors = ["Climate Modeling Alliance"]
-version = "0.6.0"
+version = "0.6.1"
```

Comment:
It looks like #51 is a breaking change? If so, this should bump the minor version.

Reply:
It removes an pre-existing default argument. So, strictly speaking, yes. I can change to `0.7.0`